src/nls/ja/strings.json

```diff
@@ -237,7 +237,7 @@
         "SECTION_COLLAPSE": ": ダブルクリックして折りたたむ",
         "SECTION_EXPAND": ": ダブルクリックして展開",
         "PANEL_COLUMN_COLLAPSE": ": クリックして折りたたむ",
-        "PANEL_COLUMN_EXPAND": ": クリックして展開",
+        "PANEL_COLUMN_EXPAND": " : クリックして展開",
```

Comment:
This doesn't seem like a whitespace problem -- it seems like an "insertion of the replacement string" problem. I can't imagine the `: ' should be in the string at all (for lines 237-240, not just 240). It looks like somehow the `":` that closes the key is getting copied to the value.

Just my $0.02...